docs/setup-cluster/on-prem/options/linux-packages.rst

```diff
@@ -118,7 +118,7 @@ Install the Determined Master and Agent
 =======================================
 
 #. Find the latest release of Determined by visiting the `Determined repo
-   <https://github.com/determined-ai/determined/releases/latest>`_.
+   <https://github.com/determined-ai/determined/releases/tag/0.27.0>`_.
```

Comment:
`https://github.com/determined-ai/determined/releases/latest` is fine. This is a case where a redirect is ok--Redirected with Found.